### PR TITLE
[UNSPLASHAPP-008] Picture / Details

### PR DESCRIPTION
**Pull request by IosBlessed**
1. In this ticket I've created picture details screen with the whole inside interaction (double tap for like/dislike, ability to zoom image)
2. Integrated Swinject and realised core modules initialisation via framework.
3. Prepared screen for interaction with Core Data

### CODE REVIEW

File: unsplahAPI/unsplahAPI/Authentification/AuthentificationCoordinatorProtocol.swift [hunk 1]
```diff
@@ -4,8 +4,10 @@
 //
 //  Created by Никита Данилович on 25.05.2023.
 //
-
+import UIKit
```
Comment (balutaeugen) [inline unsplahAPI/unsplahAPI/Authentification/AuthentificationCoordinatorProtocol.swift:7]: A bit of spacing between imports and protocols 😄 
```suggestion
import UIKit

```

File: unsplahAPI/unsplahAPI/Browse/Browse /Builder/BrowseBuilder.swift
Comment (balutaeugen) [inline unsplahAPI/unsplahAPI/Browse/Browse /Builder/BrowseBuilder.swift:1]: All the paths to `Browse` contain a weird space at the end, could you check it please?

File: unsplahAPI/unsplahAPI/Browse/PictureDetails/Protocols/PictureDetailsViewModelProtocol.swift [hunk 2]
```diff
@@ -5,4 +5,11 @@
 //  Created by Никита Данилович on 25.05.2023.
 //
 
-import Foundation
+protocol DataManager: AnyObject {
+    func extractData()
+}
```
Comment (balutaeugen) [inline unsplahAPI/unsplahAPI/Browse/PictureDetails/Protocols/PictureDetailsViewModelProtocol.swift:10]: Spacing 
```suggestion
}

```

**Review by balutaeugen — APPROVED** [review]

File: unsplahAPI/unsplahAPI/Browse/Browse /Builder/BrowseBuilder.swift
Comment (IosBlessed) [reply unsplahAPI/unsplahAPI/Browse/Browse /Builder/BrowseBuilder.swift:1]: Strange thing, I will trim them, thanks for note

File: unsplahAPI/unsplahAPI/Browse/PictureDetails/Protocols/PictureDetailsViewModelProtocol.swift (hunk 2 above)
Comment (IosBlessed) [reply unsplahAPI/unsplahAPI/Browse/PictureDetails/Protocols/PictureDetailsViewModelProtocol.swift:10]: will remove it in any case, as I don't need it. Thank you for the note!

File: unsplahAPI/unsplahAPI/Authentification/AuthentificationCoordinatorProtocol.swift (hunk 1 above)
Comment (IosBlessed) [reply unsplahAPI/unsplahAPI/Authentification/AuthentificationCoordinatorProtocol.swift:7]: thank you, will be fixed!

**Review by niclupu — APPROVED** [review]

**Pull request merged**